public/main.css

```diff
@@ -10,24 +15,94 @@ html {
 }
 
 body {
-  min-height: 100%;
-  position: relative;
+  display: flex;
+  flex-direction: column;
   margin: 0;
+  min-height: 100vh;
+  position: relative;
+}
+
+.header {
+  align-items: flex-start;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  padding: 31px;
+  width: 100%;
 }
 
 .send-logo {
+  display: flex;
   position: relative;
-  top: 31px;
-  left: 31px;
-  display: inline-block;
+  align-items: center;
+}
+
+.site-title {
+  font-size: 34px;
+  font-weight: 500;
+  margin: 0;
+  position: relative;
+  top: -1px;
+}
+
+.site-subtitle {
+  font-size: 12px;
+  margin: 0 8px;
+  color: #0c0c0d;
```

Comment:
The title and subtitle should be `#3E3D40`, and the text should be a little smaller compared to the img, according to mocks:

<img width="255" alt="image" src="https://user-images.githubusercontent.com/11432165/28528317-5be06c20-705b-11e7-8d09-a0e14fa224b5.png">